library/src/main/java/com/devbrackets/android/exomedia/core/EMListenerMux.java

```diff
@@ -111,7 +118,15 @@ public void onError(EMExoPlayer emExoPlayer, Exception e) {
     }
 
     @Override
-    public void onStateChanged(boolean playWhenReady, int playbackState) {
+    public void onStateChanged(boolean playWhenReady, @PlaybackState int playbackState) {
```

Comment:
I'm not aware of the state change inside MediaPlayer. I would have to look what the relationship between `EMVideoView` and `MediaPlayer` is. Which specific classes are you refereing to when you are saying `MediaPlayer` ?